action.yml

```diff
@@ -11,6 +11,13 @@ inputs:
   input:
     description: 'The input path.'
     default: '.'
+    required: false
```

Comment:
Not sure if we need to add this since it's the default behavior anyway, but 🤷

Reply:
I added it because https://json.schemastore.org/github-action.json incorrectly says it's required and my editor's warning was bugging me. I wonder if whatever generated that schema has a bug in it that mixed up the name "required" with the property "required". Whatever it is, I'm glad we're deprecating json and json-schema along with it.